library/oneview_version_facts.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'status': ['stableinterface'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: oneview_version
+short_description: Returns the range of possible API versions supported by the appliance.
+description:
+    - Provides an interface to return the range of possible API versions supported by the appliance.
+version_added: "2.3"
+requirements:
+    - "python >= 2.7.9"
+    - "hpOneView >= 3.2.0"
```

Comment:
The SDK on version `3.2.0` did not have this resource available.
We're currently on version `4.2.0` in the SDK, once new resources are merged (such as this one), we should likely move onto `4.3.0`, which I assume will contain this resource.